android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -95,10 +95,15 @@ public boolean onResourceReady(
             if (!(target instanceof ImageViewTarget)) {
                 return false;
             }
+
+
             ImageViewWithUrl view = (ImageViewWithUrl) ((ImageViewTarget) target).getView();
             ThemedReactContext context = (ThemedReactContext) view.getContext();
             RCTEventEmitter eventEmitter = context.getJSModule(RCTEventEmitter.class);
             int viewId = view.getId();
+            if(resource.getMinimumHeight() * resource.getMinimumWidth() < 10){
+                eventEmitter.receiveEvent(viewId, REACT_ON_ERROR_EVENT, null);
```

Comment:
not sure we want to send onLoad event then

Reply:
Then we will have same problem right? we need the load event to load always right?

Reply:
gonna quickly test, give me 5 min

Reply:
https://facebook.github.io/react-native/docs/image.html#onload

If error -> onLoadEnd should be called but not onLoad